internal/service/project/model_project.go

```diff
@@ -163,12 +163,6 @@ func NewTeamRoleList(ctx context.Context, teams []TFTeamModel) *[]admin.TeamRole
 	return &res
 }
 
-func NewGroup(tfProject *TFProjectRSModel) *admin.Group {
```

Comment:
This removal is a expected clean up right?

Reply:
yes, GroupName was splitted into Group and GroupUpdate, so  i created both NewGroup and NewUpdateGroup and then I realized NewGroup was not needed but didn't want to run all test groups again